detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/BracesOnWhenStatements.kt

```diff
@@ -0,0 +1,254 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import io.gitlab.arturbosch.detekt.api.config
+import io.gitlab.arturbosch.detekt.api.internal.Configuration
+import org.jetbrains.kotlin.com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.psi.KtBlockExpression
+import org.jetbrains.kotlin.psi.KtExpression
+import org.jetbrains.kotlin.psi.KtWhenExpression
+
+/**
+ * This rule detects `when` statements which do not comply with the specified policy.
+ * Keeping braces consistent will improve readability and avoid possible errors.
+ *
+ * Single-line `when` statement is:
+ * a `when` where each of the entries are single-line (has no line breaks `\n`).
+ *
+ * Multi-line `when` statement is:
+ * a `when` where at least one of the entries is multi-line (has a break line `\n`)
+ * or has multiple statements.
+ *
+ * Available options are:
+ *  * `never`: forces no braces on any entry.
+ *  * `necessary`: forces no braces on any entry except where necessary for multi-statement entries.
+ *  * `consistent`: ensures that braces are consistent within `when` statement.
+ *    If there are braces on one of the entry, all entries should have it.
```

Comment:
English plural rules: you can only have "one of" something, if there are multiple to begin with.
```suggestion
 *    If there are braces on one of the entries, all entries should have it.
```

Reply:
you're right